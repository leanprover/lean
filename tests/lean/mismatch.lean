definition id {A : Type} {a : A} := a
definition o : num := 1

check @id nat o
