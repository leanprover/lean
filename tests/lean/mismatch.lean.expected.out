mismatch.lean:4:7: error: type mismatch at application
  @id ℕ o
term
  o
has type
  num
but is expected to have type
  ℕ
